### PR TITLE
Hiding public types

### PR DESCRIPTION
**Pull request by WojcikMike**
Connects to https://github.com/Particular/PlatformDevelopment/issues/723

Going through public types and hiding them, for that purpose I signed ATT dll and also change dll name and default namespace name to match project name

### CODE REVIEW

**Comment by WojcikMike** [comment]
@Particular/azure-maintainers please review

File: src/Transport/CreateQueueClients.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace NServiceBus.Azure.Transports.WindowsAzureStorageQueues
     using Microsoft.WindowsAzure.Storage;
     using Microsoft.WindowsAzure.Storage.Queue;
 
-    class CreateQueueClients : ICreateQueueClients
+    class CreateQueueClients
```
Comment (SeanFeldman) [inline src/Transport/CreateQueueClients.cs]: typo in the method name "CreateRecevier()"  should be "CreateReceiver"

**Comment by SeanFeldman** [comment]
The build is red...

**Comment by WojcikMike** [comment]
@Particular/azure-maintainers build is fixed and rename was addressed

**Pull request merged**